datalad/metadata/search.py

```diff
@@ -199,14 +204,86 @@ def _get_search_schema(ds):
     return schema
 
 
-def _get_search_index(index_dir, ds, force_reindex):
+def _get_parser_autofield(idx_obj):
+    from whoosh import qparser as qparse
+
+    parser = qparse.MultifieldParser(
+        idx_obj.schema.names(),
+        idx_obj.schema)
+    # XXX: plugin is broken in Debian's whoosh 2.7.0-2, but already fixed
```

Comment:
Should there be a warning at least if broken whoosh is detected or shops we require future whoosh?

Reply:
Maybe, if we end up using that plugin at all. At present, you would have to explicitly request the 'autofield' search index in order to have this code run.